web/styles/portico/navbar.css

```diff
@@ -266,12 +274,6 @@ details summary::-webkit-details-marker {
     }
 }
 
-@media (width <= 1170px) {
-    .top-menu-tab .top-menu-tab-user-label {
-        max-width: 84px;
-    }
-}
-
```

Comment:
Should these navbar changes be extracted to a separate commit? I hadn't noticed you had navbar changes in the giant commit, and I'm not sure what these do.

Reply:
Extracted into prep commits.